src/app/public/modules/chevron/chevron.component.html

```diff
@@ -9,14 +9,32 @@
   }"
   (click)="chevronClick($event)"
 >
-  <i
-    aria-hidden="true"
-    class="sky-chevron-part sky-chevron-left"
-  >
-  </i>
-  <i
-    aria-hidden="true"
-    class="sky-chevron-part sky-chevron-right"
-  >
-  </i>
+  <ng-container *skyThemeIf="'default'">
+    <i
+      aria-hidden="true"
+      class="sky-chevron-part sky-chevron-left"
+    >
+    </i>
+    <i
+      aria-hidden="true"
+      class="sky-chevron-part sky-chevron-right"
+    >
+    </i>
+  </ng-container>
+  <ng-container *skyThemeIf="'modern'">
+    <span
+      class="chevron-glyph-container"
+    >
+      <i
```

Comment:
Not a huge deal - but we could technically pull the inner portion here out into a template that we then transclude  into both  containers. I don’t know that it is worth holding this up but would ensure that any future changes we make would only need to be made one place in the HTML.

Reply:
Good suggestion. Done!